docs/platform/how-to/create-environment.md

```diff
@@ -0,0 +1,109 @@
+# Create an environment
+
+When you [create a project](create-project.md), you'll need to create at least one [environment](../changes.md#environments). This page describes how to create an environment in more detail.
+
+There are several ways you can create a new environment, if you've not created one as part of creating the project. For example, you can do it from the top-level kebab menu:
+
+![Create environment - kebab menu](../images/how-to/create-environment/create-environment-kebab-menu.png){width=60%}
+
+You can also create an environment from the main project screen:
+
+![Create environment - main project screen](../images/how-to/create-environment/create-environment-project-screen.png){width=60%}
+
+## Name and branch
+
+You now need to give your environment a name, and select (or create) the branch it is associated with:
+
+![Name and branch](../images/how-to/create-environment/name-branch.png){width=60%}
+
+You can enter any suitable name for your environment. Examples could be: production, testing, develop, staging, prototype, and so on. Any name that suits the use case can be used.
+
+An environment is always associated with a branch. You can select a branch that already exists (such as when you have connected your project to an existing Git repository), but you are also free to create any new branches you need - simply click `+ New branch` from the branch dropdown, in the `Environment settings` dialog.
+
+## Kafka hosting option
+
+There are three hosting options:
+
+1. Quix broker
+2. Self-hosted
+3. Confluent Cloud
+
+Each of these is described briefly in the following sections.
+
+### Quix broker
+
+The simplest and most convenient choice is to enable Quix to host Kafka for you. No installation of Kafka is required, and configuration can be done through the UI if you need to change the sensible default values. Very little knowledge of Kafka is expected, beyond basic familiarity with concepts such as [topics](../glossary.md#topic).
```

Comment:
"use the Quix managed Kafka" - or something like that?

Reply:
Yes, better, my wording was a bit clumsy there.

Reply:
Changed to -> "The simplest and most convenient choice is to use Quix-managed Kafka."